src/components/GenreList.js

```diff
@@ -0,0 +1,36 @@
+
+import React, { useState } from "react";
+import '../GenreList.css';
+
+const GenreBtn = ({genre, isSelected, onGenreSelection}) => {
+    return (
+        <li>
+            <button className={"genre-item " + (isSelected ? "selected" : "")} onClick={() => onGenreSelection(genre)} >{genre} {isSelected}</button>
+        </li>
+    )
+}
+
+const GenreList = () => {
+    const [selectedGenre, setSelectedGenre] = useState('');
+
+    const genresList = ["All", "Documentary", "Comedy", "Horror", "Crime"];
+
+    function onSelect(genre) {
```

Comment:
Usage of arrow functions it is a common practice, function declaration almost never used inside component declaration
```suggestion
    const onSelect = (genre) => {
```

Reply:
@MykhailoSheliahov will be fixed in module 3